### PR TITLE
enh(ui): Use legend name in performance chart tooltip

### PR DESCRIPTION
**Pull request by bdauria**
## Description
This corrects the legend name in the chart tooltip:

Screenshot from 2020-05-04 16-19-58

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

- Open a performance chart
- Hover the time series
- The tooltip should display the data using the same name as the legend

### CODE REVIEW

File: www/front_src/src/Resources/Graph/Performance/index.tsx [hunk 1]
```diff
@@ -196,18 +205,19 @@ const PerformanceGraph = ({
           <Tooltip
             labelFormatter={formatTooltipTime}
             formatter={formatTooltipValue}
+            contentStyle={{}}
           />
```
Comment (kduret) [inline www/front_src/src/Resources/Graph/Performance/index.tsx:209]:
```suggestion
            formatter={formatTooltipValue}
          />
```

File: www/front_src/src/Resources/Graph/Performance/timeSeries.ts [hunk 2]
```diff
@@ -56,13 +56,15 @@ const getTimeSeries = (graphData: GraphData): Array<MetricData> => {
   )(graphData);
 };
 
-interface LegendColor {
-  legend: string;
+export interface LegendColor {
```
Comment (kduret) [inline www/front_src/src/Resources/Graph/Performance/timeSeries.ts:59]: This interface seems only used locally
```suggestion
interface LegendColor {
```

File: www/front_src/src/Resources/Graph/Performance/index.tsx [hunk 3]
```diff
@@ -122,8 +125,9 @@ const PerformanceGraph = ({
 
   const displayMultipleYAxes = getUnits().length < 3;
 
-  const formatValue = ({ value, unit }): string =>
-    filesize(value, { base: getBase(unit) }).replace('B', '');
+  const formatValue = ({ value, unit }): string => {
+    return filesize(value, { base: getBase(unit) }).replace('B', '');
```
Comment (kduret) [inline www/front_src/src/Resources/Graph/Performance/index.tsx:129]: to test
```suggestion
    return filesize(value, { base: getBase(unit), symbols: { B: '' } });
```

Comment (bdauria) [reply www/front_src/src/Resources/Graph/Performance/index.tsx:129]: I tried, it doesn't work unfortunately

**Comment by Thebarda** [comment]
Tested and it looks good to me 👍 

Capture d’écran (14)

**Pull request merged**